docs/interfaces.md

```diff
@@ -518,3 +518,9 @@ to storage on the device
 Can query UPower for power devices, history and statistics.
 
 * Auto-Connect: yes
+
+### usb-raw
+
+Can send and reveice data directly to any USB device attached.
```

Comment:
Typo and grammar issues. Please use: `Can directly send and receive data to an connected USB device`